src/containers/administrator/menu-item.js

```diff
@@ -18,6 +18,7 @@ class AdminMenuContainer extends Component {
 }
 
 function mapStateToProps(state, _ownProps) {
+  return {}
 }
 
 export default connect(mapStateToProps)(AdminMenuContainer);
```

Comment:
I think if `mapStateToProps` isn't doing anything then this component don't need to be connected.